packages/core/admin/admin/src/content-manager/history/pages/History.tsx

```diff
@@ -78,11 +89,11 @@ const HistoryPage = () => {
     }
   }, [versionsResponse.isLoading, navigate, query.id, versionsResponse.data?.data, query]);
 
-  if (!layout || versionsResponse.isLoading) {
+  if (isLoadingLayout || versionsResponse.isLoading) {
     return <LoadingIndicatorPage />;
   }
 
-  if (!slug || (!documentId && layout?.contentType.kind === 'collectionType')) {
+  if (!slug || (!documentId && collectionType === COLLECTION_TYPES)) {
```

Comment:
Realistically you should never run into that case, since the component is only rendered by the 2 history routes, and these are bits of data coming straight from these routes. The redirect is for extra safety (and maybe for type narrowing).

I'm adding a comment 👍

Reply:
ahhh okay, yeah I did something similar in `useDoc` where we just throw 😂